celesta.js

```diff
@@ -147,6 +148,7 @@
         enter_key_open: true,
         space_key_open: true,
         enter_key_select: true,
+        space_key_select: true,
```

Comment:
This option should be disabled by default, as such behavior for space key doesn't present on two of three major OS (Windows and Linux), appearing only on MacOSX